pbjam/priors.py

```diff
@@ -164,13 +178,14 @@ def _prior_size_check(self, pdata, numax, KDEsize):
             raise ValueError('No prior targets found within range of target. This might mean no prior samples exist for stars like this, consider increasing the uncertainty on your numax input.')
 
         elif ntgts < KDEsize:
-            warnings.warn(f'Sample for estimating KDE is less than the requested {KDEsize}.')
+            # warnings.warn recommend user change their code but logger.warning does not. Which is best here? I think the former - A. Lyttle
+            # warnings.warn(f'Sample for estimating KDE is less than the requested {KDEsize}.')
+            warnings.warn(f'Sample size for estimating KDE is {ntgts}, less than the requested {KDEsize}.')  # Add user instruction here, e.g. increase numax uncertainty?
```

Comment:
In hindsight its actually a bit misleading to say " less than the requested {KDEsize}" since when running in anything but the lowest level KDEsize can't be changed by the user. 

Maybe replace with
'...less than the suggested {KDEsize}.  The prior may not have similar stars, or if your uncertainty on numax is <~1% it may be too small.'

Reply:
I'll make a change to that. Also, should this be a `logger.warning` rather than `warnings.warn`?